src/com/mendix/widget/carousel/ui/Carousel.css

```diff
@@ -1,26 +1,26 @@
-.mx-carousel {
+.widget-carousel {
```

Comment:
Yay! But now these rules are not applied by the browser because the output classes are not updated, nor are the tests :).